package.json

```diff
@@ -10,10 +10,10 @@
     "build:codesandbox": "NODE_OPTIONS=\"--max_old_space_size=4096\" lerna run --concurrency 8 --no-private build",
     "release:version": "lerna version --no-changelog --no-push --no-git-tag-version --no-private",
     "release:build": "lerna run --concurrency 8 --no-private build --skip-nx-cache",
-    "release:changelog": "node scripts/releaseChangelog.mjs",
+    "release:changelog": "node scripts/releaseChangelog.mjs --repo base-ui",
```

Comment:
We plan to change the changelog script significantly to generate the changelog according to our format. Let's leave the local version so it's easier to change it.